theorems/T000332.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: T000332
+if:
+  P000123: true
+then:
+  P000130: true
+
+refs:
+  - mathse: 103774
+    name: Every manifold is locally compact?
+---
+
+Every point has a neighborhood that is {P123}, so homeomorphic to $\mathbb R^n$ for some non-negative integer $n$. Since $\mathbb R^n$ is {P130}, it produces a basis of compact sets.
+
+See also [ProofWiki: Locally Euclidean Space is Locally Compact](https://proofwiki.org/wiki/Locally_Euclidean_Space_is_Locally_Compact).
```

Comment:
I really don't like the style of ProofWiki, always belaboring the point and very hard to read in my opinion.  The mathse reference seems enough?